packages/amplify-cli-core/src/__tests__/help.test.ts

```diff
@@ -0,0 +1,261 @@
+import { $TSContext, CommandInfo, runHelp, commandsInfo, lookUpCommand, lookUpSubcommand, parseHelpCommands } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+
+describe('amplify help functions: ', () => {
+  printer.info = jest.fn();
+  const mockCommandsInfo: Array<CommandInfo> = [
```

Comment:
Thanks for your feedback. I think that it is a good idea to mock it so that these tests remain independent of the actual data in commandsInfo, which could change over time as features are added to or removed from the CLI.